src/main/java/build/buildfarm/common/config/Server.java

```diff
@@ -38,10 +41,22 @@ public enum INSTANCE_TYPE {
   private String publicName;
 
   public String getPublicName() {
+    // use environment override (useful for containerized deployment)
+    if (!Strings.isNullOrEmpty(System.getenv("INSTANCE_NAME"))) {
```

Comment:
Can you double check other usages of env var override to make sure we have the same order of precedence with them? I thought other configs use yaml first and env var second. This makes sense to override but just want to make sure we are consistent.

Reply:
Yeah, the precedence was not the same. It may have broken when we migrated to yaml-- I noticed the `REDIS_URI` env override did not work for the Tilt deployment as it had before.  These are fixed and consistent now. thanks!